### PR TITLE
add documentation

### PR DESCRIPTION
**Pull request by GiancarloFusiello**
In reference to https://github.com/libero/publisher/issues/272

Adds documentation about the implementation of Apache Airflow for Libero Publisher.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,10 +43,117 @@ AWS S3
...
+```
+
+### Tests
+Tests are run using [pytest](https://pytest.org/en/latest/). Test files are 
+located in the `tests/` directory. Files needed for testing, such as archives and 
+xml files, are placed in the `tests/assets/` directory. Before tests are run an
+sqlite database is used and initialized as some operations require database
+access.
+ 
+Some operators such as the `PythonOperator` expect a callable to be parsed to
```
Comment (giorgiosironi) [inline README.md]: can you add an example of another Operator (doesn't matter which one)?

Comment (GiancarloFusiello) [reply README.md]: Why?

Comment (GiancarloFusiello) [reply README.md]: This section is in reference to the context object during testing.

File: README.md [hunk 2]
```diff
@@ -34,10 +43,117 @@ AWS S3
...
+[DAG composition](https://airflow.apache.org/concepts.html#bitshift-composition).
+
+![Simple Apache Airflow Architectural Diagram](https://miro.medium.com/max/1140/1*u6duhZD2J_i1zZ0Txq26Cg.png)
+
+### DAGs
+Airflow inspects the `dags/` folder for files that instantiate the DAG class.
+Airflow expects to find this directory in the `AIRFLOW_HOME` directory.
+
+### Configuration
+Configuration files can be found in the `config/` directory. Airflow expects to
```
Comment (giorgiosironi) [inline README.md]: can you document how [environment variables can be used](https://github.com/libero/sample-configuration/blob/master/.docker/jats-ingester/airflow.cfg#L6) in there?

Comment (GiancarloFusiello) [reply README.md]: The purpose of this documentation is not to detail all of Airflows features. Those can be found in the official documentation. This documentation provides details about this specific implementation.

**Comment by giorgiosironi** [comment]
[`docker-compose` 1.23.2 CHANGELOG](https://github.com/docker/compose/blob/master/CHANGELOG.md#1232-2018-11-28):

> Reverted a 1.23.0 change that appended random strings to container names created by docker-compose up, causing addressability issues. Note: Containers created by docker-compose run will continue to use randomly generated names to avoid collisions during parallel runs.

File: README.md (hunk 1 above)
Comment (giorgiosironi) [reply README.md]: We don't use other Operators I guess, so fine as it is.

**Pull request merged**